SECURITY.md

```diff
@@ -0,0 +1,32 @@
+## Security
+
+The maintainers of this project take security seriously, and the reporting of potential security issues. If you believe you have found a security vulnerability in any CoreWCF repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
+
+We are using Microsoft's vulnerability definition as it is public and familiar to .NET users.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the project maintainers using **[security@corewcf.net](mailto:security@corewcf.net)**
+You should receive a response within 24 hours. If for some reason you do not, please follow up via email to ensure we received your original message. 
+
+Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
+
+  * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+  * Full paths of source file(s) related to the manifestation of the issue
+  * The location of the affected source code (tag/branch/commit or direct URL)
+  * Any special configuration required to reproduce the issue
+  * Step-by-step instructions to reproduce the issue
+  * Proof-of-concept or exploit code (if possible)
+  * Impact of the issue, including how an attacker might exploit the issue
+
+This information will help us triage your report more quickly.
+
+## Preferred Languages
+
+We prefer all communications to be in English.
+
+## Policy
+
+CoreWCF follows the [GitHub CVD process](https://github.blog/2022-02-09-coordinated-vulnerability-disclosure-cvd-open-source-projects/) and will use [GHSA](https://docs.github.com/en/code-security/security-advisories/about-github-security-advisories) to manage discussion of the vulnerability and fixes, and create a public CVE advisory through github if applicable.
```

Comment:
Remove the /en/ part from the docs.github.com url. This way the link will redirect to a localized version of the documentation if available.